contrib/export/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/codec"
+	extypes "github.com/cosmos/cosmos-sdk/contrib/export/types"
+	"github.com/cosmos/cosmos-sdk/contrib/export/v036"
+	"github.com/tendermint/tendermint/types"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+var (
+	migrationMap = extypes.MigrationMap{
+		"v0.36": v036.Migrate,
+	}
+	source        string
+	target        string
+	importGenesis string
+)
+
+func init() {
+	log.SetPrefix("")
+	log.SetFlags(0)
+
+	// this flag will be used once we support more than one previous version
```

Comment:
We also need the new chain-id and genesis time arguments

Reply:
Maybe I'm missing something. This is what I have at the head of the exported genesis
```
karolys-mbp:cosmos-sdk sabau$ head ../genesis_0_36.json 
{
  "genesis_time": "2019-04-22T17:00:00Z",
  "chain_id": "cosmoshub-2",
  "consensus_params": {
    "block": {
      "max_bytes": "200000",
      "max_gas": "2000000",
      "time_iota_ms": "1000"
    },
    "evidence": {

```

Reply:
They extra parameters to override the old one right?